src/invariant/test/OrderBookIsNotCrossedTests.cpp

```diff
@@ -0,0 +1,333 @@
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "invariant/OrderBookIsNotCrossed.h"
+#include "invariant/test/InvariantTestUtils.h"
+#include "ledger/LedgerTxn.h"
+#include "lib/catch.hpp"
+#include "test/TestUtils.h"
+#include "test/TxTests.h"
+#include "test/test.h"
+
+using namespace stellar;
+using namespace stellar::InvariantTestUtils;
+
+// convert a single offer into an offer operation
+Operation
+opFromLedgerEntries(LedgerEntry const& le)
```

Comment:
I don't think those two function are needed: invariants don't care about operations

Reply:
You're right, they don't -- the only loss from removing this (and the new `store()` parameter mentioned above) is that the `InvariantManager` won't report any interesting operation from an order-book-not-crossed-invariant test failure, and we'll be able to tell what the operation was by which test failed anyway, so there doesn't seem much point in having those functions.

Reply:
I've removed them.